ra-optimisation/rao-commons/src/test/java/com/farao_community/farao/rao_commons/linear_optimisation/fillers/MaxMinMarginFillerTest.java

```diff
@@ -142,17 +142,14 @@ public void fillWithMissingFlowVariables() {
         }
     }
 
-    @Test
+    @Test(expected = Test.None.class /* no exception expected */)
```

Comment:
is this normal ? 
It's just to say that this test should not throw and that it is normal if there are no assert ?

Anyway, i wouldn't be schoked if you delete this test ^^

Reply:
this test used to expect an exception when a PST was missing from the LP
I changed it to test that there is no exceptions anymore, but Sonar doesn't like it when there is no assert. So this is to say explicitly that we're just testing that there are no exceptions.

Reply:
ok, didn't know that. Thanks